understanding/info-and-relationships.html

```diff
@@ -55,212 +53,144 @@
+                     <li><a href="#brief">要約</a></li>
+                     <li><a href="#intent">意図</a></li>
+                     <li><a href="#benefits">利点</a></li>
+                     <li><a href="#examples">事例</a></li>
+                     <li><a href="#resources">関連リソース</a></li>
+                     <li><a href="#techniques">テクニック</a></li>
+                     <li><a href="#key-terms">重要な用語</a></li>
+                     <li><a href="#test-rules">テストルール</a></li>
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 1.3.1:</span>Info and Relationships (Level A)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 1.3.1:</span>情報及び関係性 (レベル A)
+
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>Information, <a href="#dfn-structure">structure</a>, and <a href="#dfn-relationships">relationships</a> conveyed through <a href="#dfn-presentation">presentation</a> can be <a href="#dfn-programmatically-determined">programmatically determined</a> or are available in text.
+                  <p>何らかの形で<a href="#dfn-presentation">提示</a>されている情報、<a href="#dfn-structure">構造</a>、及び<a href="#dfn-relationships">関係性</a>は、<a href="#dfn-programmatically-determined">プログラムによる解釈</a>が可能である、又はテキストで提供されている。
                      
                   </p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
                   
-                  <dt>Goal</dt>
-                  <dd>Information about content structure is available to more people.</dd>
+                  <dt>目標</dt>
+                  <dd>コンテンツ構造に関する情報が、より多くの人々に利用できるようにする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Use code to reinforce relationships and information conveyed through presentation.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>コードを用いて、提示を通して伝えられる関係性と情報を強化する。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>People can adapt the presentation to suit their needs while preserving the original
-                     meaning.
+                  <dt>なぜそれが重要か</dt>
+                  <dd>人々は、元の意味を維持しながら、提示を自分のニーズに合うように転用させることができる。
```

Comment:
adapt を転用とするのは違和感があります。ここでは、意味を維持したままプレゼンテーションを変更できる (= 見た目を変更しても意味が維持される) という話で、「調整」くらいではないでしょうか。

```suggestion
                  <dd>人々は、元の意味を維持しながら、提示を自分のニーズに合うように調整できる。
```